bundler/helpers/v2/run.rb

```diff
@@ -11,7 +11,7 @@
 
 require "functions"
 
-MIN_BUNDLER_VERSION = "2.0.0"
+MIN_BUNDLER_VERSION = "2.1.0"
```

Comment:
We actually rely on the Bundler v2.1.0 API for Bundler::URI, which was private in 2.0.x